### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by kathrynkodama**

### CODE REVIEW

**Comment by kathrynkodama** [comment]
@TrevCraw please review

File: README.md [hunk 1]
```diff
@@ -24,9 +25,14 @@ This project contains:
 - [Contributing](#Contributing)
 - [Feedback](#Feedback)
 --- 
+## Client IDEs
+
+Eclipse LSP4Jakarta is consumed by the following IDEs:
+-  Eclipse IDE with [Liberty Tools for Eclipse](https://github.com/OpenLiberty/liberty-tools-eclipse#welcome-to-the-liberty-tools-for-eclipse-project)
```
Comment (TrevCraw) [inline README.md:31]: should we mention that it is the early release?

File: README.md [hunk 2]
```diff
@@ -24,9 +25,14 @@ This project contains:
...
+
 ## Jakarta EE API Language Features
 
-The following is a full list of Eclipse LSP4Jakarta features offered for each [Jakarta EE API](https://jakarta.ee/specifications/). Language features will only be delivered for a given Jakarta EE API if that API is found on the project's classpath. Eclipse LSP4Jakarta is currently targeting Jakarta EE 9/9.1 (jakarta.x namespace). 
+The following is a list of Eclipse LSP4Jakarta snippets offered for each [Jakarta EE API](https://jakarta.ee/specifications/). For a full list of langauge features (snippets, diagnostics and quick-fixes) currently offered by Eclipse LSP4Jakarta see [issue #16](https://github.com/eclipse/lsp4jakarta/issues/16). Language features will only be delivered for a given Jakarta EE API if that API is found on the project's classpath. Eclipse LSP4Jakarta is currently targeting Jakarta EE 9/9.1 (jakarta.x namespace).
```
Comment (TrevCraw) [inline README.md:35]: are the current offered features just the ones that are checked off in that issue? If so, maybe mention that. (I also see some unchecked boxes next to closed issues - should those be updated as well?)

File: README.md (hunk 1 above)
Comment (kathrynkodama) [reply README.md:31]: I don't think we need to call out here that Liberty Tools for Eclipse is an early release, it is at the top of their readme

File: README.md (hunk 2 above)
Comment (kathrynkodama) [reply README.md:35]: Yes that's all the documentation we have right now until we address #350. I've checked off all the closed issues.

**Pull request merged**